errorWhenBehindPrivateImport.chpl:8: error: Bad identifier in 'except' clause, no known 'x' defined in 'B'
